library/src/main/java/ch/decent/sdk/api/AccountApi.kt

```diff
@@ -156,4 +169,59 @@ class AccountApi internal constructor(api: DCoreApi) : BaseApi(api) {
   fun createCredentials(account: String, privateKey: String): Single<Credentials> =
       getByName(account).map { Credentials(it.id, ECKeyPair.fromBase58(privateKey)) }
 
+  /**
+   * Create a transfer operation.
+   *
+   * @param credentials account credentials
+   * @param nameOrId account id or account name
+   * @param amount amount to send with asset type
+   * @param memo optional message
+   * @param encrypted encrypted is visible only for sender and receiver, unencrypted is visible publicly
+   * @param fee [AssetAmount] fee for the operation, if left [BaseOperation.FEE_UNSET] the fee will be computed in DCT asset
+   *
+   * @return a transaction confirmation
+   */
+  @JvmOverloads
+  fun createTransfer(
```

Comment:
shouldn't these methods belong to BalanceApi?

Reply:
no :)